composer.json

```diff
@@ -16,7 +16,7 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Interop\\Async\\Loop\\": "test"
+            "Interop\\Async\\Loop\\Test\\": "test"
```

Comment:
Yeah, because there's now also the Dummy-Driver, and not to be confused with the event-loop-tests repo